### PR TITLE
Added `list-workflow-runs` API

### PR DESCRIPTION
**Pull request by serkan-ozal**
# Description

Add `list-workflow-runs` API support (https://docs.github.com/en/rest/reference/actions#list-workflow-runs) to the `GHWorkflow`.

### CODE REVIEW

File: src/main/java/org/kohsuke/github/GHWorkflowRunsIterable.java [hunk 1]
```diff
@@ -18,6 +19,15 @@ public GHWorkflowRunsIterable(GHRepository owner, GitHubRequest request) {
         this.request = request;
     }
 
+    public GHWorkflowRunsIterable(GHRepository owner, GitHubRequest.Builder<?> requestBuilder) {
```
Comment (bitwiseman) [inline src/main/java/org/kohsuke/github/GHWorkflowRunsIterable.java:22]: Please make [this method](https://github.com/hub4j/github-api/blob/bcb71a3c318d243c92b757310cf153b8afc983d5/src/main/java/org/kohsuke/github/GHWorkflowRunQueryBuilder.java#L94-L99) call this constructor and remove the other constructor.

**Review by bitwiseman — CHANGES_REQUESTED** [review]
Minor change requested. 

Other than that looks great!

File: src/main/java/org/kohsuke/github/GHWorkflowRunsIterable.java (hunk 1 above)
Comment (serkan-ozal) [reply src/main/java/org/kohsuke/github/GHWorkflowRunsIterable.java:17]: Done :+1:

**Comment by serkan-ozal** [comment]
Hi @bitwiseman, when new version (`1.134`) will be released?

**Pull request merged**